src/main/scala/workflow/graph/Cacher.scala

```diff
@@ -0,0 +1,24 @@
+package workflow.graph
+
+import org.apache.spark.rdd.RDD
+import pipelines.Logging
+
+import scala.reflect.ClassTag
+
+/**
+ * Caches an RDD at a given point within a Pipeline. Follows Spark's lazy evaluation conventions.
+ *
+ * @param name An optional name to set on the cached output. Useful for debugging.
+ * @tparam T Type of the input to cache.
+ */
+case class Cacher[T: ClassTag](name: Option[String] = None) extends Transformer[T,T] with Logging {
```

Comment:
quick question: does this mean we should remove the old cacher ? the code looks identical though to the current one ?